mnist/mlcube.yaml

```diff
@@ -1,11 +1,24 @@
-schema_version: 1.0.0
-schema_type: mlcube_root
-
 name: mnist
-author: MLPerf Best Practices Working Group
-version: 0.1.0
-mlcube_spec_version: 0.1.0
+description: MLCommons MNIST MLCube example
+authors:
+  - {name: "Sergey Serebryakov", email: "sergey.serebryakov@hpe.com", org: "Hewlett-Packard Labs"}
+
+platform:
+  accelerator_count: 0
+
+container:
+  image: mlcommons/mnist:0.0.1        # Image name.
+  build_context: "mnist"              # Docker build context relative to $MLCUBE_ROOT. Default is `build`.
+  build_file: "Dockerfile"            # Docker file name within docker build context, default is `Dockerfile`.
 
 tasks:
-  - 'tasks/download.yaml'
-  - 'tasks/train.yaml'
+  download:
+    io:
+      - {name: data_dir, type: directory, io: output, default: $WORKSPACE/data}
+      - {name: log_dir, type: directory, io: output, default: $WORKSPACE/logs}
+  train:
+    io:
+      - {name: data_dir, type: directory, io: input, default: $WORKSPACE/data}
+      - {name: parameters_file, type: file, io: input, default: $WORKSPACE/parameters/default.parameters.yaml}
```

Comment:
Do we want "parameters/default.parameters.yaml" as a path, or require to just be a name, like "default.parameters.yaml".

Do we want to go flat or nested? I think we may want to require this to be flat.

Reply:
@bitfort @xyhuang @davidjurado @dfeddema 

I was thinking about how to transparently support input files, such as configuration files, with ability to have workspace in arbitrary locations (when users provide `--workspace` parameter). We need to copy files from local "workspace" folder to a different one. This may be OK. Another solution is to require parameters to be specified in a specific folder, let's say, `parameters/` inside MlCube directory. Then, MLCube will copy files from that folder to a workspace folder each time we run MLCube.